### PR TITLE
Various aesthetic updates

### PR DESCRIPTION
**Pull request by vincerubinetti**
See file diff for changes. Too many and small changes to list here.

Hold off on merging this until plugins PR is finalized, in case I make more small adjustments to this.

### CODE REVIEW

**Comment by vincerubinetti** [comment]
[will edit this comment with explanation of each change before review and merge]

File: build/themes/default.html [hunk 1]
```diff
@@ -354,57 +367,69 @@
     /* -------------------------------------------------- */
     /* when printing */
     /* -------------------------------------------------- */
-
-    /* paper/page size and margins */
+    
     @page {
-        size: letter;
```
Comment (dhimmel) [inline build/themes/default.html]: @vincerubinetti can you look at https://github.com/greenelab/manubot-rootstock/issues/105 / https://github.com/greenelab/manubot-rootstock/pull/107 to make sure removing letter won't cause a regression.

Comment (vincerubinetti) [reply build/themes/default.html]: I can't seem to find a screenshot of the problem that that issue is referencing, but I'm almost certain that the issue was caused by the poor/dirty nature of the old CSS. The `<html>`, `<body>`, and `@page` had inconsistent and nonsensical margins and paddings. Recall that `@page` had something like `margin-left: 78mm; margin-top: 56mm; margin-right: 0; margin-bottom: 0`. This was most likely the issue with bottom text getting cutoff.

The new theme should work fine for any page size. I tested it with the various ISO 216 ("A") sizes, and it seems to work. Let's revisit this at merge time when we test all the plugins and new themes thoroughly amongst several different users and manuscripts.

Comment (dhimmel) [reply build/themes/default.html]: Makes sense. I don't see why margins should only work with a single page size should things be properly implemented.

**Pull request merged**